spectec/spec/wasm-1.0/8-reduction.watsup

```diff
@@ -116,8 +116,7 @@ rule Step_read/call:
 rule Step_read/call_indirect-call:
   z; (CONST I32 i) (CALL_INDIRECT x)  ~>  (CALL_ADDR a)
   -- if $table(z, 0).ELEM[i] = a
-  -- if $funcinst(z)[a].CODE = FUNC x' (LOCAL t)* instr*
-  -- if $type(z, x) = $type(z, x')
```

Comment:
This is problematic when the function instance is an imported function.
In that case, `x'` is a type index for the imported module, but `$type(z, x')` returns a `x'`th type in the current module.

Reply:
Oops, yes.